go.mod

```diff
@@ -38,3 +38,5 @@ require (
 	knative.dev/networking v0.0.0-20210331064822-999a7708876c
 	knative.dev/pkg v0.0.0-20210331065221-952fdd90dbb0
 )
+
+replace knative.dev/eventing => github.com/cardil/eventing v0.11.1-0.20210609102549-481a26fd9454
```

Comment:
/hold

This PR requires openshift/knative-eventing#1309 to be merged first. Then this should be replaced.

Reply:
This is now removed as openshift/knative-eventing#1309 is merged

/unhold